sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultBackupAsyncClient.java

```diff
@@ -406,33 +450,56 @@ private static LongRunningOperationStatus toLongRunningOperationStatus(String op
     }
 
     /**
-     * Gets a pending {@link KeyVaultRestoreOperation full or selective restore operation} from the Key Vault.
+     * Gets a pending {@link KeyVaultRestoreOperation full restore operation} from the Key Vault.
+     *
+     * @param jobId The operation identifier.
+     * @throws NullPointerException if the {@code jobId} is null.
+     * @return A {@link PollerFlux} polling on the {@link KeyVaultRestoreOperation restore operation} status.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public PollerFlux<KeyVaultRestoreOperation, Void> beginRestore(String jobId) {
+        return beginRestore(jobId, getDefaultPollingInterval());
+    }
+
+    /**
+     * Gets a pending {@link KeyVaultRestoreOperation full restore operation} from the Key Vault.
      *
      * @param jobId The operation identifier.
+     * @param pollingInterval The interval at which the operation status will be polled for.
      * @throws NullPointerException if the {@code jobId} is null.
      * @return A {@link PollerFlux} polling on the {@link KeyVaultRestoreOperation restore operation} status.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public PollerFlux<KeyVaultRestoreOperation, Void> getRestoreOperation(String jobId) {
+    public PollerFlux<KeyVaultRestoreOperation, Void> beginRestore(String jobId, Duration pollingInterval) {
```

Comment:
I'm not sure if this conflicts with being idiomatic to Java, but the .NET guidelines specify that the operation should be hydrated from a previous operation via a public constructor rather than a parameter on the `begin...` method.

Reply:
[There is a guildeline](https://azure.github.io/azure-sdk/java_design.html#long-running-operations) for that in Java that states: "DO provide a way to instantiate a poller with the serialized state of another poller to begin where it left off, for example by passing the state as a parameter to the same method which started the operation, or by directly instantiating a poller with that state."

Does this mean we should accept a `KeyVaultBackupOperation` or even the poller object itself instead of the `jobId`? //cc @JonathanGiles, @srnagar

Reply:
That would mean those types would not be read-only anymore.

Reply:
Personally I think we should not overcomplicate things and move away from `jobId`, but I'm probably biased, maybe there's more data that backs the other option.

Reply:
The discussion we had in arch board was that Java, for now, does not offer rehydration of LROs. We have chosen to lazily add these as the demand arises, but I'm also aware that @srnagar is going to look into formalising a pattern to be used soon. For that reason my suggestion remains as it was in arch board - to not offer rehydration APIs here either.

Reply:
I was not sure where your suggestion stood on this so I went back to check the recording and got the impression these LROs were given approval after I mentioned the v-team thought we should have parity across Tier 1 languages. Thanks for clarifying that posture here. I think the best then will be removing the rehydration APIs for the time being until the guidelines are updated.

Reply:
I am working on a proposal for this and we should add this once that is finalized. Also, when this is formalized, we should add this to other libraries that have LRO APIs.